massa-bootstrap/src/tests/scenarios.rs

```diff
@@ -1,18 +1,14 @@
-// Copyright (c) 2022 MASSA LABS <info@massa.net>
+// Copyright (c) 2023 MASSA LABS <info@massa.net>
```

Comment:
this is not necessary and even harmful: 
```
The copyright notice on a work establishes a claim to copyright. The date on the notice establishes how far back the claim is made. This means if you update the date, you are no longer claiming the copyright for the original date and that means if somebody has copied the work in the meantime and they claim its theirs on the ground that their publishing the copy was before your claim, then it will be difficult to establish who is the originator of the work.
```